### PR TITLE
Add only winners and github link

### PR DESCRIPTION
**Pull request by harry48225**
image
image

Added only winners filter query parameter. Also added a button to toggle it.

### CODE REVIEW

File: site/templates/index.html [hunk 1]
```diff
@@ -6,14 +6,29 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@400;700;900&display=swap" rel="stylesheet">
+
```
Comment (BWBellairs) [inline site/templates/index.html:9]: Should we have these be local?

File: site/templates/index.html [hunk 2]
```diff
@@ -6,14 +6,29 @@
...
+      </a>
     </nav>
     
     <div class="explore">
-      Explore projects from Portfolios and hackathons
+      <span>Explore projects from Portfolios and hackathons</span>
+      <span class="filter">
+        <label for="winners">only winners</label>
+        <input type="button" class="{{ 'checked' if onlyWinners }}" id="winners" onclick="onOnlyWinnersPressed()">
+      </span>
```
Comment (BWBellairs) [inline site/templates/index.html:31]: I like your use of conditionals here

**Review by BWBellairs — APPROVED** [review]
Looks very good. Really like the design.

File: site/templates/index.html (hunk 1 above)
Comment (harry48225) [reply site/templates/index.html:9]: tbh I think its a little more hassle than its worth, doing it this way gives us a bunch of things for free.

File: site/templates/index.html (hunk 2 above)
Comment (harry48225) [reply site/templates/index.html:31]: thanks! :)

**Pull request merged**